packaging/debian/storagenode.config

```diff
@@ -187,16 +187,16 @@ case "$STATE" in
 	      if [[ ! "$diskspace" =~ ^[0-9]+[GTgt]{1} ]]; then
 		  error_ask_continue storagenode/diskspace/wrong_format
 	      else
-		  space_amount="${diskspace%*[GTgt]}"
-		  if [ "$space_amount" -lt "500" ]; then
+		  input_diskspace_bytes=`numfmt --from=iec $diskspace`
```

Comment:
Clever- I didn't even know this tool existed! And it's in coreutils, so no need to add a dependency to the package